src/device-registry/controllers/create-device.js

```diff
@@ -1,36 +1,28 @@
 const HTTPStatus = require("http-status");
 const iot = require("@google-cloud/iot");
-const isEmpty = require("is-empty");
 const client = new iot.v1.DeviceManagerClient();
 const { logObject, logElement, logText } = require("../utils/log");
-const { registerDeviceUtil } = require("../utils/create-device");
-const nearestDevices = require("../utils/nearest-device");
+const createDeviceUtil = require("../utils/create-device");
+const distance = require("../utils/distance");
```

Comment:
the distance util was enhanced with more functionalities from deleted utils whose purpose was all related to distance calculations.